package/MDAnalysis/analysis/pca.py

```diff
@@ -0,0 +1,340 @@
+element of the matrix is denoted by (i,j) row-column coordinates. The (i,j)
+coordinate reflects the influence of the of the ith frame's coordinates on the
+jth frame's coordinates of a given trajectory. The eponymous components are the
+eigenvectors of this matrix.
+
+For each eigenvector, its eigenvalue reflects the variance that the eigenvector
+explains. This value is made into a ratio. Stored in
+:attribute:`cumulat_variance`, this ratio divides the accumulated variance
+of the nth eigenvector and the n-1 eigenvectors preceding the eigenvector by
+the total variance in the data. For most data, :attribute:`explained_variance`
+will be approximately equal to one for some n that is significantly smaller
+than the total number of components, these are the components of interest given
+by Principal Component Analysis.
+
+From here, we can project a trajectory onto these principal components and
+attempt to retrieve some structure from our high dimensional data.
+
+For a basic introduction to the module, the :ref:`PCA-tutorial` shows how
+to perform Principal Component Analysis.
+
+.. _PCA-tutorial:
+The example uses files provided as part of the MDAnalysis test suite
+(in the variables :data:`~MDAnalysis.tests.datafiles.PSF` and
+:data:`~MDAnalysis.tests.datafiles.DCD`). This tutorial shows how to use the
+PCA class.
+
+First load all modules and test data ::
+    >>> import MDAnalysis as mda
+    >>> import MDAnalysis.analysis.pca as pca
+    >>> from MDAnalysis.tests.datafiles import PSF, DCD
+
+Given a universe containing trajectory data we can perform PCA using
+:class:`PCA`:: and retrieve the principal components.
+    >>> u = mda.Universe(PSF,DCD)
+    >>> PSF_pca = pca.PCA(u, select='backbone')
+    >>> PSF_pca.run()
+
+Inspect the components to determine the principal components you would like
+to retain. The choice is arbitrary, but I will stop when 95 percent of the
+variance is explained by the components. This cumulated variance by the
+components is conveniently stored in the one-dimensional array attribute
+`cumulated_variance`. The value at the ith index of `cumulated_variance` is the
+sum of the variances from 0 to i.
+
+    >>> n_pcs = next(x[0] for x in enumerate(PSF_pca.cumulated_variance)
+    >>> if x[1] > 0.95)
+    >>> atomgroup = u.select_atoms('backbone')
+    >>> pca_space = PSF_pca.transform(atomgroup, n_components=n_pcs)
+
+From here, inspection of the pca_space and conclusions to be drawn from the
+data are left to the user.
+
+Functions
+---------
+.. autoclass:: PCA
+.. autofunction:: cosine_content
+"""
+from six.moves import range
+import logging
+import warnings
+
+import numpy as np
+
+from scipy.integrate import simps
+from MDAnalysis.core import AtomGroup
+from MDAnalysis import Universe
+from MDAnalysis.analysis.align import _fit_to
+from MDAnalysis.lib.log import ProgressMeter
+
+from .base import AnalysisBase
+
+
+class PCA(AnalysisBase):
+    """Principal component analysis on an MD trajectory
```

Comment:
A short explanaition how to use the class here would be nice. SO that `help(mda.analysis.pca.PCA)` gives a short piece of code that can be run.